tree/ntuple/v7/src/RField.cxx

```diff
@@ -96,7 +96,10 @@ const std::unordered_map<std::string_view, std::string_view> typeTranslationMap{
    {"ULong_t",       "std::uint64_t"},
    {"ULong64_t",     "std::uint64_t"},
    {"unsigned long", "std::uint64_t"},
-   {"uint64_t",      "std::uint64_t"}
+   {"uint64_t",      "std::uint64_t"},
+
+   {"Long64_t",  "long long"},
+   {"ULong64_t", "unsigned long long"},
```

Comment:
I think this will overwrite the previous mappings of `Long64_t` -> `std::int64_t` and `ULong64_t` -> `std::uint64_t`!

Reply:
Ok, but then we should probably remove the other mappings, it could lead to confusion

Reply:
I'm wondering if we can replace the special case for Windows for the `(U)Long64_t` and use also `(unsigned) long long`... I'll check

Reply:
Ah yep, oversight on my part.

Reply:
The MSDN Doc says (unsigned) long long is equivalent to (unsigned) __int64

Reply:
Wait, it shouldn't matter, on Windows `(U)Long64_t` is `(unsigned) __int64`, which is equivalent to `(unsigned) long long`, on both x86 and x64, so no special case here